05_stringer/n0npax/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+// IPAddr is a type to encapsulate IPv4 (4 bytes)
+type IPAddr [4]byte
+
+// Stringer for IPAddr
+func (addr IPAddr) String() string {
+	values := make([]string, len(addr))
+	for i, v := range addr {
+		values[i] = fmt.Sprint(v)
+	}
+	return strings.Join(values, ".")
```

Comment:
This all looks good to me.  

I just wanted to point out that since you know the exact size of the array you don't need to loop it.  You could have done a one liner instead.

```
func (ip IPAddr) String() string {
	return fmt.Sprintf("%d.%d.%d.%d", addr[0], addr[1], addr[2], addr[3])
}
```

Reply:
😆 I just realised you reviewed my PR so you already know this

Reply:
refactored. Cheers :)